css/css-values/percentage-rem-low.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>REM value with :root set to low percentage</title>
+<link rel="author" title="Jon Ege Ronnenberg" href="mailto:jon.ronnenberg+wpt@gmail.com">
+<link rel="help" href="https://www.w3.org/TR/css-values-3/#rem">
+<link rel="help" href="https://www.w3.org/TR/css-fonts-3/#length-percentage-size-value">
+<link rel="match" href="../reference/ref-filled-green-100px-square-only.html">
+<link rel="mismatch" href="percentage-rem-low-ref.html">
+<meta name="assert" content="When :root font-size is 20%, a 31.25rem length must be 100px, if the browser default for :root font-size is 16px.">
+<style>
+  :root {
+    font-size: 20%; /* default is 16px in all desktop browsers */
```

Comment:
Since this be set to a fixed value in px, like 3.2px or maybe some other very small value that makes the other numbers nice and round? This way the test doesn't depend on the default font size.

Reply:
The Blink/WebKit bug is specifically about percentages, no?

That said, I'd rather we go for for 25% if we can (because then we get a nice round 4px and avoid any potential subpixel rendering differences).

Also we already assume `16px` as default all over the place. We could write a new reference (maybe a 8rem square to keep everything integral) for this if we have any interest in changing that, but I don't think we do?

Reply:
The Blink issue is not related to percentages. It's generally about minimum font-size being applied to font-size before it's used to resolve em and rem units.

Reply:
@lilles In my tests, I can always reproduce the bug if I use low percentage but not with small px values. But Trevor Burnham's (https://bugs.chromium.org/p/chromium/issues/detail?id=308862#c21) demo does highlight that it is not the only case. https://codepen.io/TrevorBurnham/pen/VXqoEW
I'm not sure why.

I don't know what a _testharness.js test_ is.

Reply:
@gsnedders I'll change it to 25%

Reply:
See https://web-platform-tests.org/writing-tests/testharness.html for `testharness.js` tests. I think writing a test based on Trevor Burnham's asserting that `window.getComputedStyle(x).fontSize` is the same for each pair and `x.offsetWidth` is the same for each pair.

Reply:
@lilles  Also the testsuite assumptions include "there is no minimum font-size" so we might not be able to test this. I wonder how much breaks (probably in the CSS2 testsuite?) if we remove that assumption…